src/test/java/org/prebid/server/it/VideobyteTest.java

```diff
@@ -0,0 +1,44 @@
+package org.prebid.server.it;
+
+import io.restassured.response.Response;
+import org.json.JSONException;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.prebid.server.model.Endpoint;
+import org.prebid.server.util.HttpUtil;
+import org.springframework.test.context.junit4.SpringRunner;
+
+import java.io.IOException;
+
+import static com.github.tomakehurst.wiremock.client.WireMock.aResponse;
+import static com.github.tomakehurst.wiremock.client.WireMock.equalTo;
+import static com.github.tomakehurst.wiremock.client.WireMock.equalToJson;
+import static com.github.tomakehurst.wiremock.client.WireMock.post;
+import static com.github.tomakehurst.wiremock.client.WireMock.urlPathEqualTo;
+import static java.util.Collections.singletonList;
+
+@RunWith(SpringRunner.class)
+public class VideobyteTest extends IntegrationTest {
+
+    @Test
+    public void openrtb2AuctionShouldRespondWithBidsFromVideobyte() throws IOException, JSONException {
+        // given
+        WIRE_MOCK_RULE.stubFor(post(urlPathEqualTo("/videobyte-exchange"))
+                .withQueryParam("source", equalTo("pbs"))
+                .withQueryParam("pid", equalTo("pubId"))
+                .withQueryParam("placementId", equalTo("placementId"))
+                .withQueryParam("nid", equalTo("nid"))
+                .withHeader(HttpUtil.ORIGIN_HEADER.toString(), equalTo("www.example.com"))
+                .withHeader(HttpUtil.REFERER_HEADER.toString(), equalTo("www.referer.com"))
```

Comment:
Please, remove this checks. Our integration tests check only for happy path, logic should be checked in unit tests.